hack/run-test-e2e-serial.sh

```diff
@@ -12,5 +26,37 @@ if [ -n "${E2E_SERIAL_FOCUS}" ]; then
 	FOCUS="-ginkgo.focus=${E2E_SERIAL_FOCUS}"
 fi
 
-echo "Running Serial, disruptive E2E Tests"
-${BIN_DIR}/e2e-serial.test ${NO_COLOR} --ginkgo.v --ginkgo.failFast --ginkgo.reportFile=/tmp/artifacts/e2e-serial --test.parallel=1 ${FOCUS}
+function setup() {
+	if [[ x"${SETUP}" != "xtrue" ]]; then
+		return 0
+	fi
+	echo "Running NRO install test suite"
+	${BIN_DIR}/e2e-install.test ${NO_COLOR} --ginkgo.v --ginkgo.failFast --ginkgo.reportFile=/tmp/artifacts/e2e-serial-install --ginkgo.focus='\[Install\] continuousIntegration'
+}
+
+function teardown() {
+	if [[ x"${TEARDOWN}" != "xtrue" ]]; then
+		return 0
+	fi
+        echo "Running NRO uninstall test suite";
+        ${BIN_DIR}/e2e-uninstall.test ${NO_COLOR} --ginkgo.v --ginkgo.reportFile=/tmp/artifacts/e2e-serial-uninstall;
```

Comment:
Also adding teardown for the ```NUMAResourcesScheduler```

Reply:
yes, I need to handle the sched part in this script